### PR TITLE
added issue api and tests

### PR DESCRIPTION
**Pull request by solankisamir**
## Description
Added support for issue api and policy in non encoded xml format
Relevant rest spec pr 
https://github.com/Azure/azure-rest-api-specs/pull/2806
https://github.com/Azure/azure-rest-api-specs/pull/2893
https://github.com/Azure/azure-rest-api-specs/pull/2881
https://github.com/Azure/azure-rest-api-specs/pull/2882

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by solankisamir** [comment]
@dsgouda @shahabhijeet @promoisha can you take a look?

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part

File: src/SDKs/ApiManagement/Management.ApiManagement/Generated/SdkInfo_ApiManagementClient.cs [hunk 1]
```diff
@@ -15,6 +15,9 @@ internal static partial class SdkInfo
                 new Tuple<string, string, string>("ApiManagement", "ApiDiagnostic", "2018-01-01"),
                 new Tuple<string, string, string>("ApiManagement", "ApiDiagnosticLogger", "2018-01-01"),
                 new Tuple<string, string, string>("ApiManagement", "ApiExport", "2018-01-01"),
+                new Tuple<string, string, string>("ApiManagement", "ApiIssue", "2018-01-01"),
```
[the remark this answers is not in the document]
Comment (solankisamir) [reply src/SDKs/ApiManagement/Management.ApiManagement/Generated/SdkInfo_ApiManagementClient.cs]: built again. there is no change in api version and no change in `.props` file

File: src/SDKs/ApiManagement/Management.ApiManagement/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -9,8 +9,8 @@
 [assembly: AssemblyTitle("Microsoft Azure API Management Resource Management Library")]
 [assembly: AssemblyDescription("Provides Api management capabilities for Microsoft Azure.")]
 
-[assembly: AssemblyVersion("4.0.0.0")]
-[assembly: AssemblyFileVersion("4.0.0.0")]
+[assembly: AssemblyVersion("4.0.1.0")]
```
[the remark this answers is not in the document]
Comment (solankisamir) [reply src/SDKs/ApiManagement/Management.ApiManagement/Properties/AssemblyInfo.cs]: reverted

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**